map.html

```diff
@@ -73,9 +73,15 @@
 </div>
 <br/>
 
-<h1 style="text-align: center;">Map Area</h1>
-<p>This is where the explanation goes.</p>
-<div style="height:80vh;" id="wrapper_Map"></div>
+
+<div style="margin-top: 1em;">
+	<button id="button_Location" title="Shows all Locations that need to be discovered.">Locations</button> 
```

Comment:
These are plain buttons, do we want to spice it up to be styled like the top bar?